nimble/host/mesh/src/access.c

```diff
@@ -401,6 +401,13 @@ static const struct bt_mesh_model_op *find_op(struct bt_mesh_model *models,
 
 		*model = &models[i];
 
+		if (BT_MESH_ADDR_IS_GROUP(dst) ||
+		    BT_MESH_ADDR_IS_VIRTUAL(dst)) {
+			if (!bt_mesh_model_find_group(*model, dst)) {
+				continue;
+			}
+		}
```

Comment:
Looks good, but should the earlier `bt_mesh_model_find_group()` call in `bt_mesh_model_recv()` be removed? It seems redundant with this addition.

Reply:
Yes, I removed that call now.